### ISSUE TITLE
When I copy location coordinates to an entry control, the format is modifed and I lost the "," character

### ISSUE THREAD

**Issue by ComptonAlvaro** [dotnet/maui#15469]
### Description

I search some place in google maps application, I copy the coordinates from a location, so I have for example 30.123123, -3.456456

When I paste this in some application like a chat in WhatsApp, I get this format, but when I copy this text to a entry control in my application, I get this: 30123123.3456456. So I lost the format of the coordinates.

I am from Spain, so in my configuration I use "," as decimal separator and "." as miles separator. Perhaps this could be the problem.

However, if this would be the problem, why in another applications I get the correct string?

How could I keep the original format?

Thanks.

### Steps to Reproduce

1. Create a project.
2. In the main page, add an entry
3. Copy the coordinates from a location in google maps application.
4. Copy it to the entry

### Link to public reproduction project repository

No needed.

### Version with bug

7.0.49

### Last version that worked well

7.0.86

### Affected platforms

Android

### Affected platform versions

Android API 33

### Did you find any workaround?

I don't find any workaround.

**Comment by IanRawley** [comment]
Is your Entry set to have a numeric keyboard?  If so, that's a known issue.  See #7582

**Comment by ComptonAlvaro** [comment]
Yes, my entry is configure as numeric, but the property of the view model is set as string.

**Comment by IanRawley** [comment]
> Yes, my entry is configure as numeric, but the property of the view model is set as string.

Then that's the problem.  As explained in the linked issue on Android an Entry with Numeric keyboard is very strict about what it will accept.  One decimal point (.) and zero commas (,) regardless of region settings, etc.  If you want to enter co-ordinates on Android you won't be able to use a Numeric keyboard on an Entry, you'll need to do something else.
